src/NuGet.Indexing/ILoader.cs

```diff
@@ -16,5 +16,7 @@ public interface ILoader
         /// <param name="config">Configuration to use.</param>
         /// <returns>Returns true if the data has changed.</returns>
         bool Reload(IndexingConfiguration config);
+
+        DateTime? GetLastUpdateTime(string name);
```

Comment:
I think this is to get the update time of the resource independent of the time it was reloaded. If for some reason the reload hasn't succeeded and we have a modified auxiliary file for sometime, this will tell us something is wrong or how long the data has remained stale. @ryuyu.